translation-service-pangeanic/src/main/java/eu/europeana/api/translation/service/pangeanic/PangeanicTranslationService.java

```diff
@@ -161,7 +161,7 @@ private void translateAndAccumulateResults(List<TranslationObj> toTranslatePerLa
   private List<TranslationObj> getObjectsWithSourceLanguage(List<TranslationObj> translationObjs,
       String sourceLanguage) {
     return translationObjs.stream()
-        .filter(to -> sourceLanguage.equals(to.getSourceLang())).toList();
+        .filter(to -> sourceLanguage.equals(to.getSourceLang())).collect(Collectors.toList());
```

Comment:
please merge the develop branch, which was updated to java 17 and revert this change